test/jasmine/tests/mapbox_test.js

```diff
@@ -475,6 +492,22 @@ describe('mapbox plots', function() {
         });
     });
 
+    it('should be able to update the access token', function(done) {
+        var promise = Plotly.relayout(gd, 'mapbox.accesstoken', 'wont-work');
+
+        promise.catch(function(err) {
+            expect(gd._fullLayout.mapbox.accesstoken).toEqual('wont-work');
+            expect(err).toEqual(new Error(constants.mapOnErrorMsg));
```

Comment:
>I'd vote :-1: for adding token validation in plotly.js. Making one request to validate the token and then one to plot the map sounds like an overkill. No?

No no no :stuck_out_tongue: I meant when the user [add a mapbox token in setting](https://github.com/plotly/streambed/pull/7105). our backend makes a request to the mapbox to validate the token. If it is not valid, they cant even add the token. So we are sure that all of the tokens in the the [token dropdown](https://github.com/plotly/streambed/pull/7160) are valid